Apps/Admin/Server/MapProfiles/DelegationChangeProfile.cs

```diff
@@ -0,0 +1,36 @@
+// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+namespace HealthGateway.Admin.Server.MapProfiles
+{
+    using AutoMapper;
+    using HealthGateway.Admin.Common.Models;
+    using HealthGateway.Database.Models;
+
+    /// <summary>
+    /// An AutoMapper profile class which defines mapping between dependent audit and delegation change models.
+    /// </summary>
+    public class DelegationChangeProfile : Profile
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DelegationChangeProfile"/> class.
+        /// </summary>
+        public DelegationChangeProfile()
+        {
+            this.CreateMap<DependentAudit, DelegationChange>()
+                .ForMember(dest => dest.DependentHdId, opt => opt.MapFrom(src => src.HdId));
```

Comment:
Related to previous suggestion:

```suggestion
            this.CreateMap<DependentAudit, DelegationChange>()
                .ForMember(dest => dest.DependentHdId, opt => opt.MapFrom(src => src.HdId))
                .ForMember(dest => dest.Reason, opt => opt.MapFrom(src => src.ProtectedReason));
```